### PR TITLE
Created a clearchat command.

### PR DESCRIPTION
**Pull request by RespDev**

### CODE REVIEW

File: src/main/java/net/cytonic/cytosis/commands/moderation/ClearchatCommand.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    public ClearchatCommand() {
+        super("clearchat", "cc");
+        setCondition((sender, _) -> sender.hasPermission("cytosis.commands.clearchat"));
+        setDefaultExecutor((sender, _) -> {
+            if (sender instanceof Player player) {
+                if (sender.hasPermission("cytosis.commands.clearchat")) {
+                    for (Player online : Cytosis.getOnlinePlayers()) {
+                        if (online.hasPermission("cytosis.commands.clearchat")) {
+                            online.sendMessage(MM."<GREEN>Chat has been cleared by \{player.getUsername()}!");
+                        }
```
Comment (Foxikle) [inline src/main/java/net/cytonic/cytosis/commands/moderation/ClearchatCommand.java:29]: Can you include the rank prefix for consistency?

**Review by Foxikle — CHANGES_REQUESTED** [review]

File: src/main/java/net/cytonic/cytosis/commands/moderation/ClearchatCommand.java (hunk 1 above)
Comment (RespDev) [reply src/main/java/net/cytonic/cytosis/commands/moderation/ClearchatCommand.java:29]: Sure

File: src/main/java/net/cytonic/cytosis/commands/moderation/ClearchatCommand.java [hunk 2]
```diff
@@ -25,7 +25,7 @@ public ClearchatCommand() {
                 if (sender.hasPermission("cytosis.commands.clearchat")) {
                     for (Player online : Cytosis.getOnlinePlayers()) {
                         if (online.hasPermission("cytosis.commands.clearchat")) {
-                            online.sendMessage(MM."<GREEN>Chat has been cleared by \{player.getUsername()}!");
+                            online.sendMessage(MM."<GREEN>Chat has been cleared by \{Cytosis.getRankManager().getPlayerRank(player.getUuid()) + player.getUsername()}!");
```
Comment (Foxikle) [inline src/main/java/net/cytonic/cytosis/commands/moderation/ClearchatCommand.java:28]: No. You need to use appending.
```java
online.sendMessage(MM."<GREEN>Chat has been cleared by".append(Cytosis.getRankManager().getPlayerRank(player.getUuid().getPrefix().append(player.getUsername())).append("!"));
```

Obviously, make sure it is syntactically correct and the styling applied properly.

**Review by Foxikle — CHANGES_REQUESTED** [review]
No this is not how it works. You need to use the append functionality. It might look like

File: src/main/java/net/cytonic/cytosis/commands/moderation/ClearchatCommand.java (hunk 2 above)
Comment (RespDev) [reply src/main/java/net/cytonic/cytosis/commands/moderation/ClearchatCommand.java:28]: ok im about to commit the patched version

**Review by Foxikle — APPROVED** [review]

**Pull request merged**